lib/jwt/AccessToken.ts

```diff
@@ -359,7 +359,9 @@ export default class AccessToken implements AccessTokenOptions {
     if (!secret) {
       throw new Error("secret is required");
     }
-    options = options || {};
+    if (!options || !options.identity) {
```

Comment:
Is this first !-check needed?

Reply:
Actually this breaks if options is undefined. https://github.com/twilio/twilio-node/actions/runs/3857410255/jobs/6574774047